homeassistant/components/light/tplink.py

```diff
@@ -39,10 +44,22 @@ def brightness_from_percentage(percent):
     return (percent*255.0)/100.0
 
 
+def rgb_to_hsv(rgb: Tuple[float, float, float]) -> Tuple[int, int, int]:
```

Comment:
They do exist in `util.color`, but the implementation is completely different. This device uses a different form of HSV (hue in degrees, saturation and value in percentages).